core/iwasm/fast-jit/fe/jit_emit_function.c

```diff
@@ -350,16 +350,19 @@ jit_compile_op_call_indirect(JitCompContext *cc, uint32 type_idx,
     arg_regs[4] = NEW_CONST(I32, func_type->param_cell_num);
     arg_regs[5] = argv;
 
-    if (!jit_emit_callnative(cc, jit_call_indirect, native_ret, arg_regs, 6)) {
+    if (jit_get_last_error(cc)
+        || !jit_emit_callnative(cc, jit_call_indirect, native_ret, arg_regs,
+                                6)) {
         return false;
     }
     /* Convert bool to uint32 */
     GEN_INSN(AND, native_ret, native_ret, NEW_CONST(I32, 0xFF));
 
     /* Check whether there is exception thrown */
     GEN_INSN(CMP, cc->cmp_reg, native_ret, NEW_CONST(I32, 0));
-    if (!jit_emit_exception(cc, JIT_EXCE_ALREADY_THROWN, JIT_OP_BEQ,
-                            cc->cmp_reg, NULL)) {
+    if (jit_get_last_error(cc)
```

Comment:
Same as above

Reply:
Thank you. This code has been deleted and found unnecessary.